spec/system/provider_interface/withdraw_an_application_at_candidates_request_spec.rb

```diff
@@ -44,6 +56,12 @@ def and_my_organisation_has_received_an_application_with_an_interview
     @interview = create(:interview, application_choice: @application_choice)
   end
 
+  def and_my_organisation_has_received_an_inactive_application_with_an_interview
+    course_option = course_option_for_provider_code(provider_code: @provider.code)
+    @application_choice = create(:application_choice, :awaiting_provider_decision, :with_completed_application_form, :inactive, course_option:)
```

Comment:
why is this using `awaiting_provider_decision` and `inactive`?

Reply:
I think the `inactive` trait also includes `with_completed_application_form` by default 🙌

Reply:
Thanks both! Copy and paste 😅